DECISIONS.md

```diff
@@ -0,0 +1,33 @@
+# Decision log
```

Comment:
DECISIONS.md: It sees like this will become redundant with decisions in Confluence.

Reply:
Good to watch out for redundancy. What appears to be emerging is that service-specific decisions live close to the code, as you see here, whereas high-level decisions affecting all or large parts of the project end up in the decision log on Confluence.